Uppsala/UppEt20.xml

```diff
@@ -0,0 +1,536 @@
+                                    <hi rend="rubric">ጸሎት፡</hi> በእንተ፡ እለ፡ ኖሙ፡ እግዚኦ፡ ግበር፡ ተዝካረ፡ ለአግብርቲከ፡ 
+                                    መሐይምናን፡ <sic corresp="JK">ወመሃይናት፡</sic> እለ፡ አዕረፉ፡ እለ፡ በልዑ፡ ሥጋከ፡ ቅዱሰ፡ 
+                                    ወደመከ፡ ክቡረ፡ አንብሮሙ፡ በየማንከ፡ ምስለ፡ አብርሃም፡ ይስሐቅ፡ 
+                                    ወያዕቆብ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                </incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.8.10">
+                                <locus from="186vb"/>
+                                <title type="complete" ref="LIT1960Mashaf#Thanksaakkweto">Prayer of incense</title>
+                                <incipit>
+                                    <hi rend="rubric">ጸሎተ፡</hi> ዕጣን፡ አአኵቶ፡ ወአሌዕሎ፡ ወእሴብሖ፡ ለእግዚአ፡ ጽድቅ፡ 
+                                    ወአሌዕል፡ ስሞ፡ ለእግዚእ፡ ንጉሠ፡ ዓለም<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                    ወእሴብሕ፡ ስሞ፡ ለአምላከ፡ ሰማይ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                    ትሴብሖ፡ ነፍስየ፡ ለእግዚአብሔር፡ ለንጉሥ፡ ዓቢይ፡ ወትትፌሣሕ፡ ቦቱ፡
+                                </incipit>
+                            </msItem>
+                            
+                                <msItem xml:id="ms_i1.8.11">
+                                    <locus from="187ra"/>
+                                    <title type="complete" ref="LIT1960Mashaf#Aakkweto"/>
+                                    <incipit>
+                                        <hi rend="rubric">ወተአኵት፡</hi> ወትብል፡ <hi rend="rubric" rendition="#partialRubric">፫</hi>ተ፡ 
+                                        ጊዜ፡ አአኵቶ፡ ለአብ፡ አአኵቶ፡ ለወልድ፡ አአኵቶ፡ ለመንፈስ፡ ቅዱስ፡
+                                    </incipit>
+                                </msItem>
+                            
+                            <msItem xml:id="ms_i1.8.12">
+                                <locus from="187ra"/>
+                                <title type="complete" ref="LIT1960Mashaf#Thanksemmesraw">Concluding prayer of thanksgiving</title>
+                                <incipit>
+                                    <locus from="187ra" to="187rb"/>
+                                    እምሥራቅ፡ ወእምዕራብ፡ <cb/>
+                                    ይትአኰት፡ ወይሴባሕ፡ ስሙ፡ ለእግዚአብሔር፡ እምአዜብ፡ ወእስከ፡ መስዕ፡ ይትአኰት፡ 
+                                    ወይ<add place="above">ሴ</add>ባሕ፡ ስሙ፡ 
+                                    ለእግዚአብሔር<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                </incipit>
+                            </msItem>
+                                
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.9">
+                                <locus target="#190va"/>
+                                <title type="incomplete" ref="LIT4327Basamay"></title>
```

Comment:
do these texts also have a sub-ID under `LIT1960Mashaf`? since ms_i1.9 is part of ms_i1 then it is preferrable that they are referred to with such sub-ID (`LIT1960Mashaf#Basamay` or similar, where `LIT1960Mashaf#Basamay` should have a corresp to `LIT4327Basamay`)

same for ms_i1.3 and 1.10

Reply:
No, they do not have sub-IDs in LIT1960Mashaf. I have no idea how fixed a part of the LIT1960Mashaf they might be. Is it preferable to create sub-IDs for them based on this singular attestation?

Reply:
cannot judge, if other LIT1960Mashaf mss do not have them then maybe let us leave it as such but at least add to LIT4327Basamay etc a relation mayFormPart